scripts/globals/roe.lua

```diff
@@ -249,7 +249,7 @@ local function completeRecord(player, record)
         type(rewards["accolades"]) == "number"
     then
         local bonusAccoladeRate = 1.0
-        bonusAccoladeRate = bonusAccoladeRate + (player:getUnityRank() or 0) * 0.05
+        bonusAccoladeRate = bonusAccoladeRate + ((player:getUnityRank() or 1) - 1) * 0.05
```

Comment:
This still doesn't resolve RoE 5, since the unity is set before this code is hit.  `CLuaBaseEntity::getUnityRank()` will only return null if no leader has been chosen, but this call happens after the first leader is set (onEventFinish in unity.lua global).

Reply:
Easy fix, move trigger:
```lua
    -- First time joining Unity (Requirements met for num Objectives and All for One set)
    if csid == zoneEventIds[zoneId][3] and option >= 1 and option <= 11 then
        changeUnityLeader(player, option)
        xi.roe.onRecordTrigger(player, 5)
        player:messageSpecial(ID.text.YOU_HAVE_JOINED_UNITY, option - 1)
```
Trigger before changing leader, and the previous logic can stand.

Reply:
Done. I approve of dealing with edge-cases within their respective areas. :+1: